site/content/en/docs/presentations/_index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Presentations"
+weight: 3.5
+description: >
+  Available online presentations about minikube
+---
+
+Presentations about the minikube project, mostly from the Kubernetes blog and the KubeCon conference.
+
+## Kubernetes Blog
+
+### 2016
+
+2016-05-30, Kubernetes 1.3, Minikube 0.1.0
+
+* _This is the initial release of Minikube._
+
+2016-07-11, Kubernetes 1.3, Minikube 0.5.0
+
+* <https://kubernetes.io/blog/2016/07/minikube-easily-run-kubernetes-locally/>
+
+### 2017
+
+### 2018
+
+### 2019
+
+2019-03-28, Kubernetes 1.14, Minikube 1.0.0
+
+* <https://kubernetes.io/blog/2019/03/28/running-kubernetes-locally-on-linux-with-minikube-now-with-kubernetes-1.14-support/>
+
+### 2020
+
+## KubeCon + CloudNativeCon
+
+### KubeCon NA 2016 (Seattle, Washington)
+
+<https://cnkc16.sched.com/?searchstring=minikube>
+
+### KubeCon EU 2017 (Berlin, Germany)
+
+<https://cloudnativeeu2017.sched.com/?searchstring=minikube>
+
+### KubeCon NA 2017 (Austin, Texas)
+
+<https://kccncna17.sched.com/?searchstring=minikube>
+
+### KubeCon EU 2018 (Copenhagen, Denmark)
+
+<https://kccnceu18.sched.com/?searchstring=minikube>
+
+* <https://www.youtube.com/watch?v=4x0CZmF_U5o> (28:30)
+  * "Minikube Intro"
+  * Dan Lorenc, Google
+
+### KubeCon NA 2018 (Seattle, Washington)
+
+<https://kccna18.sched.com/?searchstring=minikube>
+
+* <https://www.youtube.com/watch?v=2yBOVlonHQw> (38:49)
+  * "Intro: Minikube"
+  * Thomas Strömberg & Bálint Pató, Google
+
+* <https://www.youtube.com/watch?v=46-FXiSEfE4> (44:05)
+  * "Deep Dive: Minikube"
+  * Bálint Pató & Thomas Strömberg, Google
+
+### KubeCon EU 2019 (Barcelona, Spain)
+
+<https://kccnceu19.sched.com/?searchstring=minikube>
+
+* _No results found for minikube_
+
```

Comment:
while at it, lets add this one from helm summit

### Helm Summit EU 2019 (Amsterdam, The Netherlands)
* <https://www.youtube.com/watch?v=Ww7JvxKY478> (35:26)
  * "Dancing with Helm, Skaffold and Minikube! "
  *  Medya Ghazizadeh, Sr Software Engineer, Google